tests/test_ros_include_guard.py

```diff
@@ -0,0 +1,38 @@
+import pathlib
+import pytest
+from pre_commit_hooks import ros_include_guard
+
+rospkg1 = 'tests/resources/rospkg1/'
+rospkg2 = 'tests/resources/rospkg2/'
+rospkg3 = 'tests/resources/rospkg3/'
```

Comment:
I feel putting the input data and the answer close together is better.
Also, using [pytest-datadir](https://github.com/gabrielcnr/pytest-datadir) would be useful.

For example, like the following:

```
tests/ros_include_gurad/include/foo_header.hpp
tests/ros_include_gurad/include/foo_header.answer.hpp

tests/ros_include_gurad/src/bar_header.hpp
tests/ros_include_gurad/src/bar_header.answer.hpp

tests/ros_include_gurad/test/baz_header.hpp
tests/ros_include_gurad/test/baz_header.answer.hpp
```

Reply:
Same as https://github.com/tier4/pre-commit-hooks-ros/pull/32#discussion_r812330524